runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -105,6 +105,7 @@ public class ScreenshotPlugin extends Plugin
 	private static final Pattern NUMBER_PATTERN = Pattern.compile("([0-9]+)");
 	private static final Pattern LEVEL_UP_PATTERN = Pattern.compile(".*Your ([a-zA-Z]+) (?:level is|are)? now (\\d+)\\.");
 	private static final Pattern LEVEL_UP_MESSAGE_PATTERN = Pattern.compile("Congratulations, you've just advanced your ([a-zA-Z]+) level. You are now level (\\d+)\\.");
+	private static final Pattern LEVEL_UP_MESSAGE_PATTERN_99 = Pattern.compile("Congratulations, you've reached the highest possible ([a-zA-Z]+) level of 99.");
```

Comment:
You could instead update the original level up regex so that we don't need to do a separate regex match check.

```
Pattern.compile("Congratulations, you've (?:just advanced your ([a-zA-Z]+) level\\. You are now level (\\d+)|reached the highest possible ([a-zA-Z]+) level of (99))\\.");
```